app/view/templates/pages/upload/step5/_experiment_form.html

```diff
@@ -160,7 +175,11 @@ <h4>Perturbations</h4>
         </a>
       </div>
 
-      <div class="no-label flex-right">
+      <div class="flex-right help {{ "hidden" if not isPublished }} js-is-published" style="align-self: center">
+        This experiment can't be removed, since the study has been published.
+      </div>
```

Comment:
The simplest thing I decided on is to disallow removal of experiments after publishing. They can still add new ones and they can update the experiments' data, but the ids should not change, at least. I'm planning to have a "guide to publishing" in the documentation to explain how we expect publishing to go.

Reply:
Well, no, there's no way for someone to remove a study right now. We did talk about retracting a study, for which we should have a button somewhere, although even then, it should probably just show a big message on top saying it was retracted, but keep the data online.

It's possible to delete a study through the admin, of course, if we really have to (e.g. if it accidentally exposes PII)